install_sct

```diff
@@ -744,5 +744,6 @@ else
 fi
 
```

Comment:
Just leaving a common to acknowledge -- it's a bit weird that the Ubuntu 20.04 CI runners keep stalling out during the test suite!

The most recent run [on `master`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/actions/runs/1750446931) looks fine? So, maybe it was a one-off glitch. I'll try restarting the job on this PR, but... :thinking:

Reply:
Weird... I restarted the job and it worked fine. Yet, on a separate PR (https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3638), this behavior cropped up again just now! So, it looks like it might be intermittent?

Reply:
Yes, I just added a comment on #3661 with my observations.